mamutils/sendreceivemam.go

```diff
@@ -0,0 +1,87 @@
+/*
+MIT License
+Copyright (c) 2017 Harry Boer
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+//Package mamutils is a utility to create Masked Authenticated Messages (MAM's)
+package mamutils
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/iotaledger/giota"
+)
+
+//ToMAMTrytes checks its validity and casts to giota.Trytes.
+func ToMAMTrytes(t string) (tr giota.Trytes) {
+
+	trytes := ""
+	TryteValues := giota.TryteAlphabet
+
+	for _, number := range t {
+		if number > 255 {
+			//make it a space
```

Comment:
This should, at the very least, be mentioned in the description of the function. Silently converting input will just confuse people.

Reply:
Perhaps this could be documented in the README.md file?

Reply:
Add it to the description above the method and the readme if possible.